Rspec/spec/person_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../../person'
+require 'rspec/autorun'
+describe Person do
+  context 'When testing the Person class' do
+    it 'should create a new person' do
+      person = Person.new(19, 'William')
+      id = person.id
+      expect(person.age).to eq 19
+      expect(person.name).to eq 'William'
+      expect(person.id).to eq id
+    end
+  end
+end
```

Comment:
- [ ] Kindly make sure you add unit tests for all the methods in all the classes as required for the project. You can make an `it` block for each class method (at least), e.g.: **For the Person class: tests for `correct_name` method, can_use_services? method, `add_book_rentals` method,...**. The important is to make sure all the **methods** in all the classes have some tests. 
**And this should be apply in all your classes_spec files.**

Reply:
@tresorsawasawa 
Thank you for your useful feedback.
I have added the missing class which is decorator,
⚠️**Note:** we are to add tests for all classes only, not methods. 

Thank you.